site/documentation/content/admin-guide/device-connection-config.md

```diff
@@ -90,19 +90,45 @@ The server may be configured to open both a secure and a non-secure port at the
 
 ## Data Grid Connection Configuration
 
-The Device Connection component requires a connection to a remote data grid using the Infinispan Hotrod protocol to store device connection information in.
+The Device Connection component requires either an embedded cache or a remote
+data grid, using the Infinispan Hotrod protocol to store device information.
+
+The following table provides an overview of the configuration variables and corresponding command line options for configuring the common aspects of the service:
+
+| Environment Variable<br>Command Line Option | Mandatory | Default | Description                                                             |
+| :------------------------------------------ | :-------: | :------ | :-----------------------------------------------------------------------|
+| `HONO_DEVICECONNECTION_COMMON_CACHENAME`<br>`--hono.deviceConnection.common.cacheName` | no | - | The name of the cache, defaults to `device-connection` |
+| `HONO_DEVICECONNECTION_COMMON_CHECKKEY`<br>`--hono.deviceConnection.common.checkKey` | no | - | The key used to check the health of the cache. |
+| `HONO_DEVICECONNECTION_COMMON_CHECKVALUE`<br>`--hono.deviceConnection.common.checkValue` | no | - | The value used to check the health of the cache. |
```

Comment:
I believe that we do have default values for these properties, don't we? If so, we should put the values to the *Default* column IMHO ...